test-lib.test.ts

```diff
@@ -0,0 +1,57 @@
+import { newAnchorage } from './lib';
+
+describe('newAnchorage', () => {
+  test('creates a container of zeroes with specified dimensions', () => {
+    const width = 3;
+    const height = 2;
+    const container = newAnchorage(width, height);
+    expect(container.length).toBe(height);
+    container.forEach(row => {
+      expect(row.length).toBe(width);
+      expect(row.every(val => val === 0)).toBe(true);
+    });
+  });
+
+  test('creates a container of zeroes with zero width and height', () => {
+    const width = 0;
+    const height = 0;
+    const container = newAnchorage(width, height);
+    expect(container.length).toBe(height);
+  });
+
+  test('returns empty array with negative width and height', () => {
+    const width = -3;
+    const height = -2;
+    const container = newAnchorage(width, height);
+    expect(container.length).toBe(0);
+  });
+
+  test('creates a container of zeroes with large dimensions', () => {
+    const width = 1000;
```

Comment:
this test is equivalent to the one at line 4

Reply:
You are basically testing that your machine is able to allocate a lot of memory. This is not testing your application in any way - I would stick to sane values inside the unit tests and if you want to stress test it later, do an integration test on top or it with a very big data set